usagov/vote-gov-drupal
# Vote-1056: Data Test ID Updates

## Jira ticket

[Vote-1056](https://cm-jira.usa.gov/browse/VOTE-1056)

## Description

Updating data-test ids to all be camel case to provided consistency 

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. Pull down branch and run `lando retune` 
2. Cd into testing and run `npm run cy:fullTests` and verify that tests are passing 
3. Do a search for `data-test` and ensure that all test ids that are in a Cypress file or a twig template within the `votegov` theme are camel case

## Checklist for the Developer




## Checklist for the Peer Reviewers
